README.md

```diff
@@ -117,10 +117,14 @@ When the project builds, it checks the formatting and will fail if there are any
 
 ### Tools
 
-The tools that FITS bundles, such as exiftool, are downloaded and installed using the scripts in `src/main/script`.
-These scripts are run automatically as part of the `generate-resources` phase, but can also be invoked directly via
-`mvn dependency:copy@script-lib-copy exec:exec@install-exiftool`. The tools are installed into the `tools` directory.
-The tool versions and other settings are defined in the `tools.properties` file.
+Some of the tools that FITS uses are not bundled in the source tree, and are pulled in when Maven is run. They
```

Comment:
It may also be helpful to note that the FITS installation package contains all of the tool dependencies. The details noted here are only of interest to developers of FITS.

Reply:
It is under the "Development" section. Do you think the updating wording works?